client/src/styles/_common.scss

```diff
@@ -49,17 +49,60 @@ code {
 .statistics-card {
```

Comment:
I believe it might be better if all classes had a similar name so instead of `statistics-card` this could be changed to `stats-card`. We could do it the other way and change `statistics` to `stats` but I think the shorter form is clear enough.

Reply:
Changed name to `stats-card`.